python/cudf/cudf/core/series.py

```diff
@@ -2446,22 +2445,16 @@ def groupby(
         group_keys=True,
         as_index=None,
         dropna=True,
+        method="hash",
```

Comment:
can we make this `method=None` and then throw a `DeprecationWarning` if it's set to anything other than `None`?